backend-transit/src/main/java/eu/socialedge/hermes/backend/transit/domain/GoogleMapsShapeFactory.java

```diff
@@ -48,12 +46,12 @@ public GoogleMapsShapeFactory(String apiKey) {
 
     @Override
     public Shape create(List<Location> waypoints) {
-        val googleReadyWaypoints = notEmpty(waypoints).stream()
+        val waypointsLatLng = notEmpty(waypoints).stream()
             .map(l -> new LatLng(l.getLatitude(), l.getLongitude()))
             .toArray(LatLng[]::new);
 
-        List<ShapePoint> shapePoints = calculateDistanceTraveled(googleReadyWaypoints).stream()
-            .map(latLngDistTraveled -> {
+        List<ShapePoint> shapePoints = calculatePathDistanceTraveled(waypointsLatLng)
```

Comment:
Check previous comment:
unfortunately, lombok cant understand what type is here. It makes List<java.lang.Object> instead of ShapePoint.

Error:(61, 26) java: incompatible types: java.util.List<java.lang.Object> cannot be converted to java.util.Collection<eu.socialedge.hermes.backend.transit.domain.ShapePoint>